### ISSUE TITLE
height map stepping

### ISSUE THREAD

**Issue by 3diuk** [domlysz/BlenderGIS#247]
this is a 1987 x 1376 metre heightmap of buxton in the UK.  The tif downloaded is 1792 x 1536 pixels, but the data must be only 1 measurement per  170 meters.  I've previously used asc and laz files, and generally measurents are either every 1 or 2 metres.  Is this a bug with the processing?

![image](https://user-images.githubusercontent.com/22356973/74086331-c4b64800-4a79-11ea-89a1-a29cd3204028.png)

**Comment by domlysz** [comment]
SRTM data is about 90m per pixel in UE, if you want a better resolution you need to search for other data source, depending on what is available for your country, it's often not free.

**Comment by 3diuk** [comment]
wow that's small.  In the UK all lidar data is free from here:

https://environment.data.gov.uk/DefraDataDownload/?Mode=survey

it's available in various formats such as laz, las, asc etc, but I couldnt get the import option in BlenderGIS to work with any of them.

Perhaps the app could be modified to pull the height data from this website instead?  the 1 and two metre resolution would be great.

Here's another good site apparently, although I havent tested this one.

**Comment by domlysz** [comment]
Although there is currently no support for lidar files, the addon let you import any kind of digital elevation model stored in geotiff format. So it's always possible to convert lidar files to geotiff with a toolbox like LAStools. Unfortunately there is no standardized protocol for sharing elevation data across the web, so it's almost impossible to make a tools able to automatically download elevation from various service covering different country. The resolution of SRTM is low, but this dataset cover the whole world for free.

**Comment by 3diuk** [comment]
Ah too bad.  I did use lastools a few years ago to convert laz to las for houdini.  Then I wrote a houdini plugin to render a height map from the point cloud by creating a top down orthographic camera, and setting the render to be 1 pixel per point, and then assigning a grayscale vertex colour to each point based on it's z location.  After that it would render the camera to create the heightmap. 
 It was able to convert 5km square of 1metre resolution points into a heightmap around 150 times faster than houdini's standard tools.

I think this BlenderGIS is a perfect platform to implement it, particularly with EEVEE.
